examples/seq2seq/finetune_trainer.py

```diff
@@ -230,7 +233,7 @@ def main():
         freeze_params(model.get_encoder())
         assert_all_frozen(model.get_encoder())
 
-    dataset_class = Seq2SeqDataset if hasattr(tokenizer, "prepare_seq2seq_batch") else LegacySeq2SeqDataset
```

Comment:
`prepare_seq2seq_batch` is now as a function in `PretrainedTokenizer` so this cannot be `False`.

Reply:
great catch